app/views/handlers/feedback.py

```diff
@@ -114,6 +61,106 @@ def handle_post(self):
         self._session_store.session_data.feedback_count += 1
         self._session_store.save()
 
+    def get_question_schema(self):
```

Comment:
This could be defined as a cached property, which would also be consistent with the `form` property:
```python
    @cached_property
    def question_schema(self):
```

Reply:
Done, thanks!